build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
     id 'de.undercouch.download' version '5.2.1'
     id 'org.hidetake.swagger.generator' version '2.19.2'
     id 'org.sonarqube' version '4.0.0.2929'
-    id 'org.springframework.boot' version '2.7.6'
+    id 'org.springframework.boot' version '2.7.9'
```

Comment:
minor: move to 2.7.11? 
reason: Migration to 3.x recommends first having the latest 2.7 version. moving to 2.7.11 will make it easier later. Unlikely there are breaking changes between .9 and 11 

Changelog for .11 -> https://github.com/spring-projects/spring-boot/releases/tag/v2.7.11

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide